examples/snippets/modules/ROOT/examples/build-flink-streamlets-java/build.sbt

```diff
@@ -40,6 +40,7 @@ lazy val step2 = appModule("step2")
 lazy val step3 = appModule("step3")
     .enablePlugins(CloudflowFlinkPlugin)
     .settings(
+      dependencyOverrides += "com.lightbend.akka.grpc" %% "akka-grpc-runtime" % "1.0.3",
```

Comment:
since we are going to bump the version and deprecate Flink/Spark integrations this should be ok.

Reply:
Have you tested that this still works?

Reply:
The 'deprecate Flink/Spark integrations' need some nuance, since we will point users to cloudflow-contrib and use that. Is the issue with the example?

Reply:
> Have you tested that this still works in any case?

with this dependency override the tests are running and passing (haven't tested on a cluster).

>  Is the issue with the example?

The classpath